inference_models.json

```diff
@@ -192,5 +192,81 @@
       "ParameterSize": "3.8B",
       "FileSize": "2.13 GB",
       "Runtime": "directml"
+    },
+    {
+      "Name": "Phi-3-mini-4k-cuda-fp16-onnx",
+      "ProviderType": "HuggingFace",
+      "Uri": "microsoft/Phi-3-mini-4k-instruct-onnx",
+      "Path": "cuda/cuda-fp16",
+      "ModelType": "onnx",
+      "Task": "Text Generation",
+      "PromptTemplate": {
```

Comment:
May need to add `"FineTuningTemplateName": "microsoft/Phi-3-mini-4k-instruct",` otherwise user cannot click "Create Fine-tuning Project" from this model in the tree view and playground.

![image](https://github.com/microsoft/windows-ai-studio-templates/assets/9698542/aab2ea26-fce8-410a-a41c-e118e985eb6c)